src/pages/public_registration/LogPanel.tsx

```diff
@@ -44,22 +46,39 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
     staleTime: Infinity,
     gcTime: 1_800_000, // 30 minutes
   });
-
-  const userQuery = useQuery({
-    enabled: !!resourceOwnerId,
-    queryKey: ['user', resourceOwnerId],
-    queryFn: resourceOwnerId ? () => fetchUser(resourceOwnerId) : undefined,
-    retry: 0,
-    staleTime: Infinity,
-    gcTime: 1_800_000, // 30 minutes
-  });
+  const organizationAcronym = organizationQuery.isPending
+    ? ''
+    : organizationQuery.data
+      ? organizationQuery.data?.acronym
+      : t('common.unknown');
 
   const logs: LogItem[] = [];
 
+  if (isImported) {
+    registration.importDetails?.map((importDetail) => {
+      const registrationImported: LogItem = {
+        modifiedDate: importDetail.importDate,
+        title: `${t('common.importedFrom')} ${importDetail.source}`,
+        description: `${organizationAcronym}`,
+        type: 'PublishingRequest',
+      };
+      logs.push(registrationImported);
+    });
+  } else {
+    const registrationCreated: LogItem = {
+      modifiedDate: registration.createdDate,
+      title: t('common.created'),
+      description: organizationAcronym,
+      type: 'PublishingRequest',
+      actionBy: [resourceOwnerId],
```

Comment:
Hvorfor er `actionBy` et array, kan det være flere personer som har gjort en ting her? 🤔

Reply:
Ved opplastede filer (`ticketStatus = new || pending`) så kan det være forskjellige personer som har lastet opp filer, som havner på samme ticket. For enkelhets skyld så gjorde jeg bare alle om til array. Kunne sikkert satt `action: string | string[]`, men tenkte ikke på det da. Og hele komponenten kommer til å skrives om når jeg implemeterer det nye designet, så har ikke gått tilbake for å endre 😇 

<img width="515" alt="image" src="https://github.com/BIBSYSDEV/NVA-Frontend/assets/10809754/f3974eea-8fe3-4ba8-bdfe-4c6bfe0f622c">

Reply:
Ah crap, skjønner. Når jeg ikke trodde loggen hadde flere problemer 😅